### ISSUE TITLE
Adding Lean 4

### ISSUE THREAD

**Issue by markusschmaus** [KevinBatdorf/code-block-pro#247]
Lean 4 has a great VS Code integration. Is it possible to include its syntax highlighting in this plugin?

https://marketplace.visualstudio.com/items?itemName=leanprover.lean4

**Comment by KevinBatdorf** [comment]
Yeah I'll look into it this weekend or if I have time sooner. It could take some time to get merged in though as I have to submit it upstream to the rendering library and wait for them to tag a release. I'll update here though.

**Comment by KevinBatdorf** [comment]
Added here: https://github.com/shikijs/shiki/pull/527

As soon as they tag a release I'll add it to the plugin. Will follow up when that happens
